.github/workflows/production.yml

```diff
@@ -0,0 +1,50 @@
+name: CI_CD
+
+on:
+  push:
+    branches: [main]
```

Comment:
triggered on push of main branch or workflow_dispatch allows us to run it anytime.